src/prpl.ts

```diff
@@ -51,6 +67,8 @@ export function makeHandler(root?: string, config?: Config): (
   console.info(`Serving files from "${absRoot}".`);
   const builds = loadBuilds(absRoot, config);
   const cacheControl = (config && config.cacheControl) || 'max-age=60';
+  const unregisterMissingServiceWorkers =
+      (config && config.unregisterMissingServiceWorkers === false) || true;
```

Comment:
Looks like `unregisterMissingServiceWorkers` is always evaluated to `true` here.

Reply:
Oops! Fixed and added test.